pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -140,12 +140,12 @@
     ConsumerBuilder<T> topicsPattern(Pattern topicsPattern);
 
     /**
-     * Specify a pattern for topics that this consumer subscribes on.
+     * Specify a pattern for topics that this consumer subscribes to.
      *
-     * <p>It accepts regular expression that is compiled into a pattern internally. Eg.
+     * <p>Accepts a regular expression that is compiled into a pattern internally. Eg.
```

Comment:
```suggestion
     * <p>It accepts a regular expression that is compiled into a pattern internally. E.g.,
```

I don't think changing it to a third-person verb is a good idea. IMO, it makes the annotation even more complex, especially when it's inconsistent throughout the file.